src/pages/order-item/tabs/tests/add-product-form-tests/add-product-form.spec.js

```diff
@@ -78,4 +105,30 @@ describe('tests for add-product-form component', () => {
     expect(addBtn).toHaveProperty('disabled', false);
     expect(mockSetFieldValue).toHaveBeenCalledTimes(1);
   });
+  it('shoud handle change in input', () => {
+    const input = screen.getByTestId('promo-input');
+    const value = 'wallet';
+    fireEvent.change(input, { target: { value } });
+    expect(input).toHaveValue(value);
+  });
+  it('clear input after checking promoCode', async () => {
+    const input = screen.getByTestId('promo-input');
+    const value = 'wallet';
+    const button = screen.getByTestId('promo-button');
+    fireEvent.change(input, { target: { value } });
+    fireEvent.click(button);
+
+    expect(await screen.findByTestId('promo-input')).toHaveValue('');
+  });
+  it('should show error after entering wrong promoCode', async () => {
+    const input = screen.getByTestId('promo-input');
+    const value = 'wall';
+    const button = screen.getByTestId('promo-button');
+    const error = 'Промокод не знайдено';
```

Comment:
I would advise you against using hard-coded values, you could just import an error constant you created.